tests/rptest/tests/write_caching_fi_test.py

```diff
@@ -200,7 +200,9 @@ def test_unavoidable_data_loss(self):
 
         # Crash the second node. Since it didn't fsync the data, the data
         # some messages will be locally lost on the next restart.
-        self.redpanda.signal_redpanda(self.redpanda.nodes[1])
+        second_node = self.redpanda.nodes[1]
+        self.redpanda.signal_redpanda(second_node)
```

Comment:
I was looking at the code.. can we use redpanda.stop_node(node, forced=true).. I think that does what you are looking for.

Reply:
Hm. Let me try. I didn't see it before.

Reply:
Done. Thanks for the tip! 🙇‍♂️